swap_meet/__init__.py

```diff
@@ -0,0 +1,8 @@
+conditions = {
```

Comment:
Nice use of `dict` for this! O(1) lookup vs worst case O(n) for n conditions for an `if/else` to fall through to the last condition!